generators/client/templates/angular/src/main/webapp/app/shared/social/_social.component.ts

```diff
@@ -21,7 +21,9 @@ export class <%=jhiPrefixCapitalized%>SocialComponent implements OnInit {
     ) {}
 
     ngOnInit() {
+        <%_ if (enableTranslation) { _%>
```

Comment:
this is not required as there is a pre processing that would strip this based on regex. The actual fix would be to change the line below to `this.languageService.addLocation(['social']);` so that its detected by the regex